### PR TITLE
GitHub Add Get PRs Of Branch Command

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
Enhancement. No related issue.

## Description
Adding command to get PRs corresponding to given branch name.

## Screenshots
Because GitHub Test Playbook is skipped. I tested my command on a small playbook to assure it runs as expected in Demisto (Also have been performing other checks).
Adding photo of the related small playbook and the success.
Tom-Github-Command-Test_Wed_Apr_28_2021
Tom-Github-Command-Test_Wed_Apr_28_2021 (1)

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/GitHub/Integrations/GitHub/GitHub.py [hunk 1]
```diff
@@ -616,6 +617,52 @@ def create_pull_request_command():
...
+    response = http_request('GET', url_suffix=suffix)
+    formatted_outputs = [format_pr_outputs(output) for output in response]
+
+    return CommandResults(
+        outputs_prefix='GitHub.PR',
+        outputs_key_field='Number',
+        outputs=formatted_outputs,
+        raw_response=response,
+        readable_output=tableToMarkdown(f'Pull Request For Branch #{branch_name}', formatted_outputs, removeNull=True)
+    )
```
Comment (ShahafBenYakir) [inline Packs/GitHub/Integrations/GitHub/GitHub.py:644]: Let's move this into the list_branch_pull_requests_command function

File: Packs/GitHub/Integrations/GitHub/GitHub.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import demistomock as demisto
 from CommonServerPython import *
-from CommonServerUserPython import *
+
+# from CommonServerUserPython import *
```
Comment (ShahafBenYakir) [inline Packs/GitHub/Integrations/GitHub/GitHub.py:4]:
```suggestion
from CommonServerUserPython import *
```

File: Packs/GitHub/Integrations/GitHub/GitHub_test.py [hunk 3]
```diff
@@ -15,9 +14,13 @@ def load_test_data(json_path):
         return json.load(f)
 
 
+branch_pull_requests_data = load_test_data('./test_data/get-branch-pull-requests-response.json')
```
Comment (ShahafBenYakir) [inline Packs/GitHub/Integrations/GitHub/GitHub_test.py:17]: Why this isn't in `test_list_branch_pull_requests_command`?

**Review by ShahafBenYakir — COMMENTED** [review]
Nice!

File: Packs/GitHub/Integrations/GitHub/GitHub_test.py (hunk 3 above)
Comment (tomneeman151293) [reply Packs/GitHub/Integrations/GitHub/GitHub_test.py:17]: will change name

**Review by ShahafBenYakir — APPROVED** [review]

**Comment by tomneeman151293** [comment]
@ShahafBenYakir All steps have succeeded, validation only fails on update docker image as we decided not to update.

**Comment by tomneeman151293** [comment]
@roysagi  @amshamah419 Could any of you do the doc review?

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 4]
```diff
@@ -2806,6 +2806,567 @@ script:
     - contextPath: GitHub.File.DownloadUrl
       description: Link to download the file content.
       type: String
+  - arguments:
+    - default: false
+      description: The branch name to retrieve its pull requests.
```
Comment (amshamah419) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:2811]:
```suggestion
      description: The branch name to retrieve its pull requests.
```
```suggestion
      description: The branch name from which to retrieve pull requests.
```

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 5]
```diff
@@ -2806,6 +2806,567 @@ script:
...
+      secret: false
+    - default: false
+      description: The name of the organization.
+      isArray: false
+      name: organization
+      required: false
+      secret: false
+    - default: false
+      description: The repository of the pull request. Defaults to repository parameter
+        if not given.
```
Comment (amshamah419) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:2824]:
```suggestion
      description: The repository for the pull request. Defaults to the repository parameter
        if not provided.
```

File: Packs/GitHub/ReleaseNotes/1_2_3.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### GitHub
+- Added **GitHub-list-branch-pull-requests** command.
```
Comment (amshamah419) [inline Packs/GitHub/ReleaseNotes/1_2_3.md:4]:
```suggestion
- Added the ***GitHub-list-branch-pull-requests*** command.
```

**Pull request merged**